src/components/home/home.tsx

```diff
@@ -53,20 +112,62 @@ const Home = () => {
           </Row>
           <Row className="mb-4 mt-2 justify-content-center">
             <Container className="w-75">
-              <Form.Select className="d-inline-block w-25">
-                <option>Study Type</option>
-              </Form.Select>
-              <Form.Select className="d-inline-block w-25">
-                <option>Type</option>
-              </Form.Select>
-              <Form.Select className="d-inline-block w-25">
-                <option>Size</option>
-              </Form.Select>
+              <div className="w-75">
+                <Form.Select
```

Comment:
Both the Form.Select code can be moved to a method and can be called for each of the facet type

Reply:
Once the Form.Select block is converted to a method, we can call it as many times needed even if there are more than 2 different facet types. The code is the same for both the facet types, just pass the key of the facet to the method as argument.